nat/traversal/pinger.go

```diff
@@ -101,6 +106,7 @@ type natProxy interface {
 type Params struct {
 	RequestConfig json.RawMessage
 	Port          int
+	Cancel        chan struct{}
```

Comment:
I don't know, but `Cancel` channel does not look like a `traversal.Params`. Maybe something wrong with such structure?